djangocms_blog/models.py

```diff
@@ -171,7 +172,8 @@ class Post(KnockerModel, ModelMeta, TranslatableModel):
 
     translations = TranslatedFields(
         title=models.CharField(_('title'), max_length=255),
-        slug=models.SlugField(_('slug'), max_length=255, blank=True, db_index=True),
+        slug=models.SlugField(_('slug'), max_length=255, blank=True,
```

Comment:
A bit wary about adding another dependency for such a small feature. Can we just pass the argument only in 1.9 and above?